scripts/commands/fafnir.lua

```diff
@@ -60,7 +60,7 @@ function onTrigger(player)
     -- Use the mob object as you normally would
     mob:setSpawn(player:getXPos(), player:getYPos(), player:getZPos(), player:getRotPos())
 
-    mob:setDropID(0) -- No loot!
```

Comment:
I think the reason this used setDropID was to server as an example of how to give a dynamic mob a drop list via script, since it would not have any sql to do so with. so this should stay for that reason.

Reply:
Added it back in